packages/storybook/src/stories/Icon.stories.tsx

```diff
@@ -114,13 +114,13 @@ export const all: StoryObj = {
         </span>
         <span className="icon-story">
           <Icons.Logout />
```

Comment:
Let's rename the object name as well.  These should always match to avoid confusion.

Reply:
Let's make sure that is reflected in the Jira story too. I'll update the Jira ticket.